tests/test_api_v1.py

```diff
@@ -493,8 +495,18 @@ def test_set_state(mock_rmtree, mock_exists, state, app, client, db, worker_auth
     assert fetched_request['state_history'][0]['state_reason'] == state_reason
     assert fetched_request['state_history'][0]['updated']
     assert fetched_request['state_history'][1]['state'] == 'in_progress'
-    mock_exists.assert_called_once_with('/tmp/cachito-archives/bundles/temp/1')
-    mock_rmtree.assert_called_once_with('/tmp/cachito-archives/bundles/temp/1')
+    mock_exists.assert_called_once_with(
+        os.path.join(
+            tempfile.gettempdir(),
+            'cachito-archives/bundles/temp/1'
+        )
+    )
+    mock_rmtree.assert_called_once_with(
+        os.path.join(
+            tempfile.gettempdir(),
+            'cachito-archives/bundles/temp/1'
```

Comment:
Please assign the value of `os.path.join(tempfile.gettempdir(), 'cachito-archives/bundles/temp/1')` to a variable above so that this value is not computed twice.

Reply:
resolved in https://github.com/release-engineering/cachito/pull/97/commits/a8c9c326f258c1dadacaca0ea989b4da725b36c6